ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -0,0 +1,26 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-tests
+  commands: GOFLAGS="-mod=readonly" make test
+  container:
+    from: src
+- as: func-tests
+  commands: go mod tidy && go mod vendor && make functest
+  container:
+    from: bin
```

Comment:
I don't think we'll be successful in running the functional tests like this. For now i think we can just execute the unit tests until we get a func test suite that can execute on openshift ci.

Reply:
I agree
I removed the func tests, The func tests will be added later